source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -966,6 +993,11 @@ See the `Load Balancer Specification <../load-balancers/load-balancers.rst#event
        *   - "connectionError": The Connection check out attempt experienced an error while setting up a new Connection
        */
       reason: string|Enum;
+
+      /**
+       * See `ConnectionCheckedOutEvent.duration`.
+       */
```

Comment:
I understand the desire to reduce duplication but can we include the full description here?  All other events have duplicate comments on duplicate fields, and it's annoying to have to jump around the file manually to see what this field contains (especially since it's not a link, just text).

Reply:
> it's annoying to have to jump around

I argue that it is more annoying to try and figure out whether `ConnectionCheckOutFailedEvent.duration` is the same thing as `ConnectionCheckedOutEvent.duration` or not by comparing two large pieces of text. It will also be annoying when those pieces need to be changed, especially when they diverge due to a human error because there is no mechanism keeping them consistent.

Given the above, do you still require the documentation to be duplicated?